int32 robot_id  # ロボットのID
float32 radius  # 射出するフェロモンの半径
